ui/js/content.js

```diff
@@ -290,10 +290,10 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
      * @see manageaccess#initManageAccess
```

Comment:
When sharing a collaborative doc as an editor, there should be a dropdown in the "share" overlay (similar to manage access) where you can specify the role (can edit or can view)